.github/workflows/proto-registry.yml

```
@@ -0,0 +1,92 @@
+name: Buf Push
+# Protobuf runs buf (https://buf.build/) push updated proto files to https://buf.build/provenance-io
+# This workflow is only run when a .proto file has been changed or when this workflow has been modifed.
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "**/**.proto"
+      - ".github/workflows/proto-registry.yml"
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"     # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+-rc*" # Push events to matching v*, i.e. v1.0-rc1, v20.15.10-rc5
```

```suggestion
on:
  push:
    branches:
      - main
    paths:
      - "third_party/**.proto"
      - ".github/workflows/proto-registry.yml"
```

See other comment for reasoning.

---

done.